aten/src/ATen/functorch/BatchRulesUnaryOps.cpp

```diff
@@ -167,6 +167,14 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   UNARY_POINTWISE(special_i1);
   UNARY_POINTWISE(special_i1e);
   UNARY_POINTWISE(special_ndtri);
+  UNARY_POINTWISE(special_bessel_j0);
+  UNARY_POINTWISE(special_bessel_j1);
+  UNARY_POINTWISE(special_bessel_y0);
+  UNARY_POINTWISE(special_bessel_y1);
+  UNARY_POINTWISE(special_modified_bessel_i0);
+  UNARY_POINTWISE(special_modified_bessel_i1);
+  UNARY_POINTWISE(special_scaled_bessel_k0);
+  UNARY_POINTWISE(special_scaled_bessel_y1);
```

Comment:
I don't think these are the right operators. The valid arguments to UNARY_POINTWISE are the functions defined in https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml, which are slightly different from the torch.* APIs (the torch.* apis end up calling one or more of these operations)

Reply:
Thank you so much for pointing that out to me, I really appreciate it! Looking in the file linked, when I searched for `special_bessel_j0`, I got `func: special_bessel_j0(Tensor self) -> Tensor` on line 13778, and `func: special_bessel_j0.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)` on line 13784. I noticed that the former calls the latter, and I was wondering if I should call the latter directly on line 170 in my PR. Thank you so much!